terraform/modules/configs/deploy-all/main.tf

```diff
@@ -302,6 +302,21 @@ resource "aws_security_group_rule" "s3-virus-scan-allow-outgoing" {
   cidr_blocks       = ["0.0.0.0/0"]
 }
 
+resource "aws_security_group" "s3-virus-scan-lambda" {
+  vpc_id      = data.aws_ssm_parameter.vpc_id.value
+  name        = "s3-virus-scan-lambda-${lower(var.stage)}"
+  description = "Allow inbound db traffic"
+}
+
+resource "aws_security_group_rule" "s3-virus-scan-lambda-allow-http" {
+  type              = "egress"
+  from_port         = 4567
+  to_port           = 4567
+  protocol          = "tcp"
+  security_group_id = aws_security_group.s3-virus-scan-lambda.id
+  cidr_blocks       = [data.aws_vpc.scale.cidr_block]
+}
+
```

Comment:
Hi @tmckernan  - this Security Group/Rule can also be moved into the s3-virus-scan-service as it is only accessed from within the service (the others were at this level as the are either used in multiple modules or as a hangover from the initial port of TF code from Sparks who had them here - if they can be moved into modules then that is preferable).

If you could make that change and the arn -> name one below - I can give it another run on SBX1 (I did it earlier - with that change in place - hooked up to a test S3 bucket - and it all worked) - so just this last tidy up and should be good to go 👍